src/SourcemapToolkit.SourcemapParser/SourceMapGenerator.cs

```diff
@@ -56,10 +56,23 @@ public MappingGenerateState(IList<string> names, IList<string> sources)
 
 	public class SourceMapGenerator
 	{
-		/// <summary>
-		/// Serialize SourceMap object to json string with given serialize settings
-		/// </summary>
-		public string SerializeMapping(SourceMap sourceMap, JsonSerializerSettings jsonSerializerSettings = null)
+        /// <summary>
+        /// Convenience wrapper around SerializeMapping, but returns a base 64 encoded string instead
+        /// </summary>
+        public string SerializeMappingBase64(SourceMap sourceMap, JsonSerializerSettings jsonSerializerSettings = null)
```

Comment:
Or alternatively, don't make this method prepend the comment. I'm not sure what other scenarios this would be helpful for besides this comment string